lib/config/yml_generator_config.dart

```diff
@@ -250,8 +252,12 @@ class YmlGeneratorConfig {
   void checkIfTypesAvailable() {
     final names = <String>{};
     final types = <String>{};
+    final extend = <String>{};
```

Comment:
extends is a strange name for a set, I would use extendedNames or extendedModels. extend does not indicate that it is a list.